xchange-stream-okcoin/src/main/java/info/bitrich/xchangestream/okcoin/dto/okx/OkxStreamingAuthenticator.java

```diff
@@ -0,0 +1,34 @@
+package info.bitrich.xchangestream.okcoin.dto.okx;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.stream.Collectors;
+
+public class OkxStreamingAuthenticator {
```

Comment:
should probably use https://github.com/knowm/XChange/blob/develop/xchange-core/src/main/java/org/knowm/xchange/service/BaseParamsDigest.java

Reply:
Fixed.